openquake/hazardlib/tests/geo/surface/gc2_test.py

```diff
@@ -112,10 +112,10 @@ def _setup_peer_test_bending_fault_config():
         Line([PNT1, PNT2, PNT3, PNT4, PNT5]), usd, lsd, dip, mesh_spacing)
```

Comment:
Here, mesh_spacing is still used as parameter

Reply:
Yes, but it is a parameter of `SimpleFaultSurface.from_fault_data` and I have removed it only from PlanarSurface.